matomo/index.js

```diff
@@ -32,13 +32,12 @@ module.exports = function (context) {
-            _paq.push(['trackPageView']);
```

Comment:
Is this ok to remove?

Reply:
Yes, because the trackPageView event was firing twice the first time a page rendered. We're already doing the tracking on L13 in matomo/track.js file.